contracts/access/Whitelist.sol

```diff
@@ -13,81 +13,81 @@ import "../access/rbac/RBAC.sol";
 contract Whitelist is Ownable, RBAC {
 
   // Name of the whitelisted role.
-  string private constant ROLE_WHITELISTED = "whitelist";
+  string private constant _ROLE_WHITELISTED = "whitelist";
```

Comment:
I think we should make an exception for constants. :slightly_smiling_face:

Reply:
Oh, no, why? if there are public constants for some reason, they will not have the underscore. Then you can't tell what's private constant and what is not.

Reply:
Hm, okay. We can postpone this decision anyway.